### ISSUE TITLE
Getting "Method not found" error

### ISSUE THREAD

**Issue by SimonFletcher** [TurnerSoftware/InfinityCrawler#67]
Hi,

I'm trying to use InfinityCrawler in an Azure Function built on DotNet Core 3.1.  I've basically dropped the example code in, but am getting an exception when it runs:
```
"Method not found: 'System.Threading.Tasks.Task`1<TurnerSoftware.RobotsExclusionTools.RobotsFile> TurnerSoftware.RobotsExclusionTools.RobotsFileParser.FromUriAsync(System.Uri)'."
```
I added the library using: 
`dotnet add package InfinityCrawler --version 0.3.0`

Am I missing something, or is there a compatibility issue?

Thanks!

**Comment by Turnerj** [comment]
Hey, thanks for raising the issue! I think the problem is an update I made to my RobotsExclusionTools library that InfinityCrawler depends on - I believe I broke binary compatibility when I added methods with default parameter values (specifically having `CancellationToken cancellationToken = default`) without then recompiling Infinity Crawler to target the newer versions... or something like that.

Either way, I've released a new version of Infinity Crawler which has various dependency updates including that dependency. Try installing `0.4.0` when it works its way onto NuGet shortly.
